### DISCUSSION TITLE
xtts2 or similar model, training from scratch?

### DISCUSSION THREAD

**Discussion by danielw97** [idiap/coqui-ai-TTS#175] [Q&A] [answered]
Hi,
First off, many thanks for continuing work on this project.
These tts resources are extremely valuable, so it's great we are still able to use them.
I'm currently trying to gain knowledge in this field, and one of my strategies is training some small models on my own ljspeech formatted dataset.
I'm interested in experimenting with these, and am just wondering if anyone has developed anything to train xtts from scratch as apposed to finetuning?
Failing that, is there a similarly advanced model that folks recommend working/experimenting with?
Thanks in advance for any pointers or tips anyone is willing to share.

**Comment by eginhard** [comment, marked as answer]
The full XTTS training code was never released, you can only fine-tune it. Anything else also doesn't make sense if you don't have thousands of hours of data.

**Comment by danielw97** [comment]
Thanks for the answer, that's about what I figured.
